tests/golden/defaults/openshift4-authentication/openshift4-authentication/10_oauth.yaml

```diff
@@ -5,4 +5,31 @@ metadata:
   labels:
     name: cluster
   name: cluster
-spec: {}
+spec:
+  identityProviders:
+    - ldap:
+        bindDN: uid=service,ou=idp,dc=company,dc=tld
+        bindPassword:
+          name: company-ldap
+        ca:
+          name: company-ldap
+        url: ldaps://ldap.company.tld:636/ou=services,dc=company,dc=tld?uid
+      name: Company LDAP
+      type: LDAP
+    - mappingMethod: add
+      name: keycloak-auth
+      openID:
+        claims:
+          email:
+            - email
+          name:
+            - name
+          preferredUsername:
+            - preferred_username
+        clientID: company-client
+        clientSecret:
+          name:
+            clientSecret: t-silent-test-1234/c-green-test-1234/keycloak-auth/clientSecret
+        clientSecretRef: oidc-client
```

Comment:
This parameter isn't needed in the final output. Is this a problem? Can this key be removed? Are there alternatives?